RiotSwiftUI/Modules/Room/PollHistory/View/PollListItem.swift

```diff
@@ -36,20 +38,76 @@ struct PollListItem: View {
             Text(pollData.formattedDate)
                 .foregroundColor(theme.colors.tertiaryContent)
                 .font(theme.fonts.caption1)
-
+            
             HStack(alignment: .firstTextBaseline, spacing: 8) {
                 Image(uiImage: Asset.Images.pollHistory.image)
                     .resizable()
                     .frame(width: imageSize, height: imageSize)
-
+                
                 Text(pollData.question)
                     .foregroundColor(theme.colors.primaryContent)
                     .font(theme.fonts.body)
                     .lineLimit(2)
                     .accessibilityLabel("PollListItem.title")
             }
+            
+            if pollData.winningOption != nil {
+                VStack(alignment: .leading, spacing: 12) {
+                    optionView(winningOption: pollData.winningOption!)
+                    resultView
+                }
+            }
         }
     }
+    
+    private var clipShape: some Shape {
+        RoundedRectangle(cornerRadius: 4.0)
+    }
+
+    private func optionView(winningOption: TimelinePollAnswerOption) -> some View {
+        VStack(alignment: .leading, spacing: 12.0) {
+            HStack(alignment: .top, spacing: 8.0) {
+                Text(pollData.winningOption!.text)
+                    .font(theme.fonts.body)
+                    .foregroundColor(theme.colors.primaryContent)
+                    .accessibilityLabel("PollListData.winningOption")
```

Comment:
Feels like this should be an accessibilityIdentifier rather

Reply:
Definitely!
Applied -> https://github.com/vector-im/element-ios/pull/7278/commits/01baa0718e9424ebc084055ad3b516a5075e3d33